### PR TITLE
[closes #104] feat: new crop - Jalapeño

### LINKED ISSUE #104: new crop: Jalapeño
I think this would be a good next addition because it has a really distinct shape that would work well with the game's art style, and also would work well with some of the other available crops for new recipes such as Salsa (onion + tomato + corn + jalapeño), or spicy queso (cheese + jalapeño).

proposed growth stages: 1 days seed, 2 days sprout (leafy), 2 days growing pepper. 5 days total
proposed tier: 2

### PR DESCRIPTION
**Pull request by lstebner**
adds new crop to all necessary places and image files

I started with the soybean files and then tweaked them just to get something working for testing purposes. feel free to change these all you want @jeremyckahn , but hopefully having a starting point is helpful!

Screen Shot 2021-06-14 at 4 09 12 PM

### CODE REVIEW

File: src/data/items.js [hunk 1]
```diff
@@ -325,6 +326,31 @@ export const tomato = crop({
...
+    [GROWING]: 3,
+  },
+  growsInto: 'jalapeno',
+  id: 'jalapeno-seed',
+  name: 'Jalapeño Seed',
+  tier: 4,
+})
+
+/**
+ * @property farmhand.module:items.carrot
```
Comment (jeremyckahn) [inline src/data/items.js:346]: This needs to be changed to `@property farmhand.module:items.jalapeno`.

**Review by jeremyckahn — CHANGES_REQUESTED** [review]
This is looking really good! Thanks for putting all this together. I left one note of a JSDoc annotation to fix up, but otherwise the code looks great (and thanks for reordering the other crops)!

I've also tweaked the art a bit, what do you think?

Screen Shot 2021-06-16 at 8 21 23 AM

File: src/data/items.js (hunk 1 above)
Comment (lstebner) [reply src/data/items.js:346]: oops, caught me doing some copypasta 😅 . thanks for catching that

**Comment by lstebner** [comment]
those look excellent @jeremyckahn ! great work! looks like we're about ready to merge this (i have a feature-jalapeno branch) and then make some recipes

**Review by jeremyckahn — APPROVED** [review]
LGTM! 🌶👌

**Pull request merged**